distribution/lib/Standard/Base/0.0.0-dev/src/Enso_Cloud/Internal/Existing_Enso_Asset.enso

```diff
@@ -98,7 +98,7 @@ type Existing_Enso_Asset
     is_regular_file self = self.asset_type == Enso_Asset_Type.File
 
     ## PRIVATE
-    list_directory self =
+    list self =
```

Comment:
This was internal API so IMO did not have to be renamed.

But renaming seems OK too. Feels like `list` is nice for consistency, whereas `list_directory` is nice for explicit-ness in the internal API. Both have a + and - so either seems fine.